web/src/lib/static/loadDisplayContent.test.ts

```diff
@@ -14,36 +14,6 @@ describe("loadDisplayContent", () => {
     mockedFs = fs as jest.Mocked<typeof fs>;
   });
 
-  describe("loadRoadmapSideBarDisplayContent", () => {
-    it("returns sidebar card content from markdown", () => {
-      const welcomeCard =
-        "---\n" +
-        "id: welcome-id\n" +
-        "header: Welcome!\n" +
-        "imgPath: /img/congratulations-purple.svg\n" +
-        "color: accent-cooler\n" +
-        "headerBackgroundColor: accent-cooler-lightest\n" +
-        "hasCloseButton: false\n" +
-        "weight: 1\n" +
-        "---\n" +
-        "**Welcome!**";
-
-      mockReadDirReturn(["welcome.md"]);
-      mockedFs.readFileSync.mockReturnValue(welcomeCard);
-
-      expect(loadRoadmapSideBarDisplayContent().sidebarDisplayContent["welcome-id"]).toEqual({
-        id: "welcome-id",
-        header: "Welcome!",
-        imgPath: "/img/congratulations-purple.svg",
-        color: "accent-cooler",
-        headerBackgroundColor: "accent-cooler-lightest",
-        hasCloseButton: false,
-        weight: 1,
-        contentMd: "**Welcome!**",
-      });
-    });
-  });
```

Comment:
[pebble] This should not have been deleted. We still need to test that `loadRoadmapSideBarDisplayContent` displays properly. And it doesn't matter that this is called "welcome" because it is mocking the read file & read dir. It is never actually accessing the real welcome card, it could be mocking a file named anything, it doesn't matter.

Reply:
placed back